dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/monitor/DatadogRumMonitor.kt

```diff
@@ -31,6 +32,8 @@ internal class DatadogRumMonitor(
     private val userInfoProvider: UserInfoProvider = NoOpUserInfoProvider()
 ) : RumMonitor {
 
+    private val allViewEvents = ConcurrentHashMap<UUID, RumEvent>()
```

Comment:
I don't see where you actually clear this. We need to clear it from time to time ....maybe when the event is closed ?

Reply:
No, that's the point. Because the resource will be stopped after the view is stopped, we need to keep the view.
When we refactor this bit we can eventually have something slightly better (the resource will have a link to it's parent scope meaning we'll only keep what's still in use)